ClientHandler.py

```diff
@@ -15,4 +14,8 @@ def Network_message(self, data):
 
     def Message(self, data):
         print(self, "Sending message")
-        self.Send({"action": "message", "message": data})
\ No newline at end of file
+        self.Send({"action": "message", "message": data})
+
+    def Network_newBombFromPlayer(self, data):
+        for p in self._server.players:
+            p.Send({"action": "bombFromServer", "bomb": data["bomb"]})
```

Comment:
This is wrong, because of my previous PR, where I delete WeakKeyDictionary to have only array of dictionaries. You should use only playersInfoArray["channel"] in order to send info to players